packages/mdl-checkbox/mdl-checkbox.scss

```diff
@@ -18,9 +18,6 @@
 @import "./variables";
 @import "./keyframes";
 
-// TODO: Docs
-// TODO: x-browser test!
-
 @function mdl-checkbox-transition($property, $timing-function, $delay: 0ms, $duration: $mdl-checkbox-transition-duration) {
```

Comment:
I'm OK with small changes like this, as long as we don't think it's a problem for our commit strategy, particularly where it comes to automatic changelogs.